doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+.. _quantization:
+
+Quantization
+============
+
+nGraph Quantization 
+-------------------
+
+Quantization refers the process of reducing the number of bits that represent a
+number. In the context of deep learning, weights and activations can be
+represented using 8-bit integers (or INT8) to compress the model size of a
+trained neural network, without any significant loss in model accuracy. Compared
+with 32-bit floating point (FP32), using arithmetic with lower precision such as
+INT8 to calculate weights and activation requires less memory. However, memory
+reduction varies based on the model (ResNet 50, Inception v3, BERT, etc.).
+
+Implementing a quantized model with nGraph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To implement a quantized model with
+nGraph, provide a partially (or fully) quantized FP32 model (e.g., where the
+convolution layer in the model is replaced with a quantized convolution) to the
+nGraph library along with quantized parameters: weights, activations, scale, and
+zero point. 
+
+.. Note:: Currently, nGraph only supports quantization for inference.
+
+nGraph Quantized Operators (Ops)
+--------------------------------
+
+nGraph uses scale and zero point (also used by ONNX) to map real values to
+quantized values.  All quantized ops (APIs) in nGraph use scale and zero point
+and they can be used just like any other nGraph op. 
+
+**Scale**: the quantization scale for the tensor 
+
+**Zero point**: the zero point of the tensor 
+
+**Round mode**: used in combination with scale and zero point to round real 
+values to quantized values
+
+.. table:: nGraph Quantized Ops
+
+
+	+----------------------+-----------------------------------------------+
+	| Operator             | Description                                   |
+	+======================+===============================================+
+	| Quantize             | Maps real values (r) to quantized values (q)  |
+	|                      | using scale (s), zero point (z),              |
+	|                      | and round mode; produces a quantized tensor   |
```

Comment:
Minor nit:  you use sentence-casing (end with period) on the other table but not this one.